samples/sampler/stories/default/Icon.js

```diff
@@ -34,15 +39,22 @@ const SkinnedIcon = Skinnable(
 			}
 		}
 
+		const iconType = select('icon type', ['glyph', 'url src', 'custom'], Config, 'glyph');
+		let children;
+
+		switch (iconType) {
+			case 'glyph': children = select('icon', ['', ...iconNames], Icon, 'home'); break;
+			case 'url src': children = select('src', ['', docs, factory, logo], Config, logo); break;
+			default: children = text('custom icon', Config);
```

Comment:
`custom icon` and `icon type` is not a prop of Icon, but developers can think that they are icon prop.
I think it's better to provide a knob for `children`.

Reply:
Changed `iconType` with `children`. This way a developer should be aware that an icon can have a string as children(glyph) or a path to a .png or .svg image.

Reply:
I didn't know that your previous code corresponded with  Sandstone.
It seems to have been inevitable to provide various types of children. I'll revert it to the previous code.